templates/books-index.html

```diff
@@ -0,0 +1,75 @@
+<div class="container">
+  <br>
+  <div class="row">
+    <div class="col-md-3 col-sm-4 col-xs-6 grid-div" ng-repeat="book in booksIndexCtrl.books | orderBy: 'title'">
+      <div class="thumbnail">
+        <a ng-href='/books/{{book._id}}'><img class='img-responsive book-img' ng-src="{{book.image}}" alt="book image"/></a>
+          <div class="book-title text-center">
+            {{book.title}}
+        </div>
+      </div>
+    </div>
+  </div>
+
+<div class='row'>
+  <div class='col-md-1 col-sm-1 col-md-offset-12 addBtn-div'>
+    <button ng-click="adding=!adding" class='btn btn-default add-shadow'>
+      <span class='glyphicon glyphicon-plus' aria-hidden='true'></span>
```

Comment:
do you use this glyphicon for anything? it seems like it's just leftover from copy/pasting.